src/clr-angular/utils/conditional/if-active.service.spec.ts

```diff
@@ -41,20 +41,11 @@ export default function(): void {
         expect(currentChange).toEqual(11);
       });
 
-      it('sets the current active value in the service', function() {
+      it('tracks the current active value in the service', function() {
         expect(ifActiveService.current).not.toBeDefined();
         ifActiveService.current = 5;
         expect(ifActiveService.current).toEqual(5);
       });
-
-      it('gets the current active value of the service', function() {
-        const init = 1;
```

Comment:
Why are we removing this test?

Reply:
This is a duplicate of one several lines above.

Reply:
Kind of, I guess. But this test seemed to test whether the initialized value showed up. I don't know if it's important to test that or not here.